parser/parser.go

```diff
@@ -861,6 +863,21 @@ func (p *Parser) parseIndexExpression(left ast.Expression) ast.Expression {
 	return exp
 }
 
+// parseIndexExpression parse an array-index expression.
```

Comment:
Comment here refers to the wrong method-name.

Reply:
Thanks! I had not been aware of this detail.